packages/editor/src/components/document-outline/test/index.js

```diff
@@ -78,7 +78,10 @@ describe( 'DocumentOutline', () => {
 		} );
 
 		it( 'should not render when no heading blocks provided', () => {
-			const blocks = [ paragraph ];
+			const blocks = [ paragraph ].map( ( block, index ) => {
+				// Set client IDs to a predictable value.
+				return { ...block, clientId: '_clientId_' + index };
```

Comment:
Any reason to lead with the underscore here?

```suggestion
				return { ...block, clientId: `clientId_${ index }` };
```